# Configuration for Crichton.

# Torso
torso_state: "state-torso"
torso_ref:   "ref-torso"
torso_mode:  "SNS_MOTOR_MODE_VEL"
torso_max_vel:   0.3
torso_max_accel: 0.1
torso_map:
  {
  crichton_torso_joint: 0
  }

# Left arm
left_arm_state: "state-left"
left_arm_ref:   "ref-left"
left_arm_mode:  "SNS_MOTOR_MODE_VEL"
left_arm_max_vel:   0.4
left_arm_max_accel: 0.1
left_arm_map:
  {
  crichton_left_1_joint: 0,
  crichton_left_2_joint: 1,
  crichton_left_3_joint: 2,
  crichton_left_4_joint: 3,
  crichton_left_5_joint: 4,
  crichton_left_6_joint: 5,
  crichton_left_7_joint: 6
  }

# Right arm
right_arm_state: "state-right"
right_arm_ref:   "ref-right"
right_arm_mode:  "SNS_MOTOR_MODE_VEL"
right_arm_max_vel:   0.4
right_arm_max_accel: 0.1
right_arm_map:
  {
  crichton_right_1_joint: 0,
  crichton_right_2_joint: 1,
  crichton_right_3_joint: 2,
  crichton_right_4_joint: 3,
  crichton_right_5_joint: 4,
  crichton_right_6_joint: 5,
  crichton_right_7_joint: 6
  }

# Left hand
left_hand_state: "sdhstate-left"
left_hand_ref:   "sdhref-left"
left_hand_mode:  "SNS_MOTOR_MODE_POS"
left_hand_max_vel:   1.0
left_hand_max_accel: 0.5
left_hand_map:
  {
  crichton_left_knuckle_joint: 0,
  crichton_left_finger_12_joint: 1,
  crichton_left_finger_13_joint: 2,
  crichton_left_thumb_2_joint: 3,
  crichton_left_thumb_3_joint: 4,
  crichton_left_finger_22_joint: 5,
  crichton_left_finger_23_joint: 6
  }

# Right hand
right_hand_state: "sdhstate-right"
right_hand_ref:   "sdhref-right"
right_hand_mode:   "SNS_MOTOR_MODE_POS"
right_hand_max_vel:   1.0
right_hand_max_accel: 0.5
right_hand_map:
  {
  crichton_right_knuckle_joint: 0,
  crichton_right_finger_12_joint: 1,
  crichton_right_finger_13_joint: 2,
  crichton_right_thumb_2_joint: 3,
  crichton_right_thumb_3_joint: 4,
  crichton_right_finger_22_joint: 5,
  crichton_right_finger_23_joint: 6
  }
